docs/configuration/git-providers/monitoring-conf/loki-values.yaml

```diff
@@ -0,0 +1,24 @@
+resources:
```

Comment:
The path for this file seems quite strange ` docs/configuration/git-providers/monitoring-conf/loki-values.yaml`

Why is this under "git-providers" ?

Reply:
Yes, it was wrong. Thanks. I'm moving it elsewhere, in keeping with @pauloribeiro-codacy 's suggestion.